pkg/virt-api/webhooks/mutating-webhook/mutators/vmi-mutator_test.go

```diff
@@ -414,42 +414,46 @@ var _ = Describe("VirtualMachineInstance Mutator", func() {
 		),
 	)
 
-	DescribeTable("should add the default network interface",
-		func(iface string) {
-			expectedIface := "bridge"
-			switch iface {
-			case "masquerade":
-				expectedIface = "masquerade"
-			case "slirp":
-				expectedIface = "slirp"
-			}
+	Context("should add network interfaces", func() {
+		var kvConfig *v1.KubeVirt
 
-			permit := true
-			testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, &v1.KubeVirt{
+		BeforeEach(func() {
+			kvConfig = &v1.KubeVirt{
 				Spec: v1.KubeVirtSpec{
 					Configuration: v1.KubeVirtConfiguration{
-						NetworkConfiguration: &v1.NetworkConfiguration{
-							NetworkInterface:     expectedIface,
-							PermitSlirpInterface: &permit,
-						},
+						NetworkConfiguration: &v1.NetworkConfiguration{},
 					},
 				},
-			})
+			}
+		})
+
+		It("should add the bridge network interface", func() {
+			kvConfig.Spec.Configuration.NetworkConfiguration.NetworkInterface = "bridge"
+			testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, kvConfig)
 
 			_, vmiSpec, _ := getMetaSpecStatusFromAdmit(rt.GOARCH)
-			switch expectedIface {
-			case "bridge":
-				Expect(vmiSpec.Domain.Devices.Interfaces[0].Bridge).NotTo(BeNil())
-			case "masquerade":
-				Expect(vmiSpec.Domain.Devices.Interfaces[0].Masquerade).NotTo(BeNil())
-			case "slirp":
-				Expect(vmiSpec.Domain.Devices.Interfaces[0].Slirp).NotTo(BeNil())
-			}
-		},
-		Entry("as bridge", "bridge"),
-		Entry("as masquerade", "masquerade"),
-		Entry("as slirp", "slirp"),
-	)
+			Expect(vmiSpec.Domain.Devices.Interfaces[0].Bridge).NotTo(BeNil())
+		})
+
+		It("should add the masquerade network interface", func() {
+			kvConfig.Spec.Configuration.NetworkConfiguration.NetworkInterface = "masquerade"
+			testutils.UpdateFakeKubeVirtClusterConfig(kvInformer, kvConfig)
+
+			_, vmiSpec, _ := getMetaSpecStatusFromAdmit(rt.GOARCH)
+			Expect(vmiSpec.Domain.Devices.Interfaces[0].Masquerade).NotTo(BeNil())
+		})
+
+		It("should add the slirp network interface", func() {
+			permit := true
```

Comment:
Maybe you can use `pointer.P(true)` instead.

Reply:
Done